policy/modules/kernel/kernel.te

```diff
@@ -347,6 +347,7 @@ files_list_root(kernel_t)
 files_list_etc(kernel_t)
 files_getattr_etc_runtime_dirs(kernel_t)
 files_mounton_etc_runtime_dirs(kernel_t)
+files_mounton_runtime_dirs(kernel_t)
```

Comment:
is that `/run/initramfs`?

Reply:
Yes, though I will probably remove this access since after removing the selinux dracut module, it seems this is no longer needed.

Reply:
Removing the SELinux dracut module in my testing environment solved this denial.